client/src/app/core/actions/is-sorting-payload.ts

```diff
@@ -0,0 +1,5 @@
+import { TreeIdNode } from 'app/core/ui-services/tree.service';
+
+export interface IsSortingPayload {
```

Comment:
`Is...` is the naming for functions checking, if an object implements the interface (e.g. https://github.com/OpenSlides/openslides-client/blob/cddc802cee1adf796f4170200aab26012311d32c/client/src/app/core/core-services/data-store-upgrade.service.ts#L13) . So this should be `SortingPayload`.